packages/docusaurus-theme-classic/src/theme/CodeBlock/CopyButton/index.tsx

```diff
@@ -7,6 +7,7 @@
 
 import React, {useCallback, useState, useRef, useEffect} from 'react';
 import clsx from 'clsx';
+// @ts-expect-error: TODO, we need to make theme-classic have type: module
 import copy from 'copy-text-to-clipboard';
```

Comment:
This is specifically because Webpack resolution is neither `node` nor `NodeNext`. Webpack allows `exports`, importing ESM in CJS, importing CJS in ESM, using both `require` and `import` in one file, importing `.ts` extension, importing `.js` extension... everything.

https://twitter.com/atcb/status/1536831060960419841